openwisp_notifications/types.py

```diff
@@ -111,3 +111,10 @@ def _unregister_notification_choice(notification_type):
             NOTIFICATION_CHOICES.pop(index)
             return
     raise ImproperlyConfigured(f'No such Notification Choice {notification_type}')
+
+
+def get_notification_choices():
```

Comment:
Are you sure this is necessary?

I don't see the django docs saying that passing a callable is necessary for [choice fields](https://docs.djangoproject.com/en/5.2/ref/models/fields/#choices). Am I missing something?

Reply:
I have added the comment in the code to explain the implementation details of Django 

From the docs: 

> Passing a callable for choices can be particularly handy when, for example, the choices are:
> 
> - the result of I/O-bound operations (which could potentially be cached), such as querying a table in the same or an external database, or accessing the choices from a static file.
> 
> - a list that is mostly stable but could vary from time to time or from project to project. Examples in this category are using third-party apps that provide a well-known inventory of values, such as currencies, countries, languages, time zones, etc.

We fall in the second point. The `NOTIFICATION_CHOICES` changes when registering/unregistering notification types.

Reply:
Good, point I was forgetting this important detail for a moment.